test/spec/index.spec.ts

```diff
@@ -214,6 +214,18 @@ describe('html to image', () => {
         .catch(done)
     })
 
+    it('should render slot-ed elements', (done) => {
+      Helper.bootstrap(
+        'web-components-sloted/node.html',
+        undefined,
+        'web-components-sloted/image',
+        'web-components-sloted/sc.js',
+      )
+        .then((node) => htmlToImage.toPng(node))
+        .then(Helper.check)
```

Comment:
This test is passing on mac and failing on Linux. Seems that the src for the 2 images is the same, anyone encountered something like this?